cmd/oras/tag/tag.go

```diff
@@ -42,19 +49,24 @@ func TagCmd() *cobra.Command {
 Example - Tag the manifest 'v1.0.1' in 'locahost:5000/hello' to 'v1.0.2':
   oras tag localhost:5000/hello:v1.0.1 v1.0.2
 
-Example - Tag the manifest with digest sha256:9463e0d192846bc994279417b50114606712d516aab45f4d8b31cbc6e46aad71 to 'v1.0.3'
-  oras tag localhost:5000/hello@sha256:9463e0d192846bc994279417b50114606712d516aab45f4d8b31cbc6e46aad71 v1.0.3
+Example - Tag the manifest with digest sha256:9463e0d192846bc994279417b50114606712d516aab45f4d8b31cbc6e46aad71 to 'v1.0.2'
+  oras tag localhost:5000/hello@sha256:9463e0d192846bc994279417b50114606712d516aab45f4d8b31cbc6e46aad71 v1.0.2
+
+Example - Tag the manifest 'v1.0.1' in 'locahost:5000/hello' to 'v1.0.2' with the custom concurrency number of 1:
+  oras tag --concurrency 1 localhost:5000/hello:v1.0.1 v1.0.2
 `,
-		Args: cobra.ExactArgs(2),
+		Args: cobra.MinimumNArgs(2),
 		PreRunE: func(cmd *cobra.Command, args []string) error {
 			return opts.ReadPassword()
 		},
 		RunE: func(_ *cobra.Command, args []string) error {
 			opts.srcRef = args[0]
-			opts.targetRef = args[1]
+			opts.targetRef = args[1:]
 			return tagManifest(opts)
 		},
 	}
+
+	cmd.Flags().Int64VarP(&opts.defaultTagNOptions.Concurrency, "concurrency", "", 0, "provide concurrency number, default is 5")
```

Comment:
If the default is `5`, please make it `5`.
```suggestion
	cmd.Flags().Int64VarP(&opts.defaultTagNOptions.Concurrency, "concurrency", "", 5, "concurrency level")
```
so that it decouples with the `oras-go` library and the default value will be printed in `oras tag --help`.

Reply:
done